feedhenry/fh-android-sdk
# Rhmap2761 2739

# What this fixes
 * RHMAP 2761 and RHMAP 2739  

# How to test
 * Connect Android device or emulator
 * ./gradlew clean connectedCheck 
 * Build should pass  

# Bonus testing points 
 * Add this to the template and make sure it works.
